includes/Admin/Settings_Screens/Product_Sync.php

```diff
@@ -318,6 +318,27 @@ public function get_settings() {
 			],
 			[ 'type' => 'sectionend' ],
 
+			[
+				'name' => __( 'Feed File Settings', 'woocommerce' ),
+				'type' => 'title',
+				'id'   => 'feed_file_settings',
+			],
+
+			[
+				'id'      => \WC_Facebookcommerce_Integration::SETTING_FEED_FILE_GENERATION_ENABLED,
+				'title'   => __( 'Enable feed file generation.', 'facebook-for-woocommerce' ),
+				'desc'    => sprintf(
+					/* translators: url to documentation section. */
+					__( 'Feed file is used for cyclic catalog content synchronization. Please check the documentation at %s to learn if this option should be enabled for your store.', 'facebook-for-woocommerce' ),
```

Comment:
Assuming we stick with a checkbox in UI for this, let's iterate on the wording to make it clear what we recommend. We can also use a tooltip to declutter the user interface.